`git-https`-like feature that uses reqwest backend in gix?

Would you consider adding a feature to support https-based URLs, but using a reqwest backend, rather than a curl backend?

Concretely, I am asking whether the following could be added to Cargo.toml's [`features` section](https://github.com/frewsxcv/rust-crates-index/blob/d588ccf6a828efdd303de24e2828545ffaf9439f/Cargo.toml#L75):
```toml
## Like `git-https` but using a reqwest backend instead of curl
git-https-reqwest = ["git", "gix?/blocking-http-transport-reqwest"]
```

For context, I have a project that uses both rust-crates-index and [rustsec](https://github.com/rustsec/rustsec/tree/main/rustsec). Rustsec uses gix with the reqwest backend. As a result, when I try to build both together, I get the error below.

Between the two, rustsec seems like it would be more difficult to change, which is why I have come here.
```
error[E0428]: the name `Impl` is defined multiple times
   --> .../.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/gix-transport-0.45.0/src/client/blocking_io/http/mod.rs:220:1
    |
217 | pub type Impl = curl::Curl;
    | --------------------------- previous definition of the type `Impl` here
...
220 | pub type Impl = reqwest::Remote;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `Impl` redefined here
    |
    = note: `Impl` must be defined only once in the type namespace of this module

error: Cannot set both 'http-client-reqwest' and 'http-client-curl' features as they are mutually exclusive
  --> .../.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/gix-transport-0.45.0/src/client/blocking_io/http/mod.rs:26:1
   |
26 | compile_error!("Cannot set both 'http-client-reqwest' and 'http-client-curl' features as they are mutually exclusive");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -84,6 +84,8 @@ git-performance = ["git", "gix?/max-performance"]
 ##
 ## Most will want this enabled unless they know to only have to access non-https URLs for their indices.
 git-https = ["git", "gix?/blocking-http-transport-curl"]
+## Like `git-https` but using a reqwest backend instead of curl
+git-https-reqwest = ["git", "gix?/blocking-http-transport-reqwest"]
 ## Allow some functions to receive `rayon`-powered siblings for higher performance.
 parallel = ["dep:rayon"]
 ## Add support for communicating with sparse indices.

